C/ds/Preordertraversal.c

```diff
@@ -0,0 +1,100 @@
+#include<stdio.h>
+#include<conio.h>
+#include<stdlib.h>
+
+struct BST{
+ int data;
+ struct BST *left;
+ struct BST *right;
+};
+
+struct BST* create();
+void insert(struct BST*, struct BST*);
+void preorderRecursive(struct BST*);
+void preorderIterative(struct BST*);
+
+int main(){
```

Comment:
Display message before & after of  inserting element in the node

Reply:
Added display message.